pandas/_libs/lib.pyx

```diff
@@ -1562,7 +1562,7 @@ def infer_datetimelike_array(arr: ndarray[object]) -> str:
                 seen_tz_aware = True
 
             if seen_tz_naive and seen_tz_aware:
-                return 'mixed'
+                return "mixed"
```

Comment:
cc @MarcoGorelli would not be averse to a pre-commit to make sure quoting is uniform per file (e.g. either ' or "), ideally we should just be uniform across the entire codebase.

Reply:
black ensures we use double in the python code; the cython is a mishmash but mostly double at this point